### PR TITLE
Adding Radio, Checkbox, and Button HTML and CSS into master

### PR DESCRIPTION
**Pull request by orinevares**
- created primary button HTML and CSS file and copied code from Atom into it
- created checkbox HTML and CSS file and copied code from Atom into it
- created radio HTML and CSS file and copied code from Atom into it

### CODE REVIEW

File: components/primary button/sample.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
+    <title>Primary Button</title>
+  </head>
+  <body>
+
+<button class="primarybutton" type="button" name="button">Submit</button>
```
Comment (patricksimonian) [inline components/primary button/sample.html]: indentation for button

File: components/Checkbox/style.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/* Customize the label (the container) */
+.checkbox {
+  display: block;
+  position: relative;
+  padding-left: 25px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-family: 'PT Sans';
```
[the remark this answers is not in the document]
Comment (orinevares) [reply components/Checkbox/style.css]: I added the fall back font families.

The font family is in the checkbox class so is that not already at the higher level and cascading down?

**Comment by jleach** [comment]
@orinevares Are you able to fix the capitalization of the `Checkbox` folder from the web UI? Same question for the "primary button" folder name.

**Comment by orinevares** [comment]
@jleach doesn't seem like it. Are you able to do so now that it is merged into the master?

**Comment by jleach** [comment]
Ya, I'll just create a separate PR for the change. Will do shortly.

**Comment by patricksimonian** [comment]
@orinevares @jleach Sorry, i totally missed that and merged. Yes can we make another PR to fix the titles to the folder names. In addition i think the "primary button" folder should follow a programmy naming convention, either snake_case or camelCase ie 'primary_button' or 'primaryButton'

**Comment by jleach** [comment]
Also I noticed that it creates a hidden directory called `.github`. Any reason for it being hidden?

**Comment by patricksimonian** [comment]
hmm thats interesting. .github is special configuration directory for things like issue templates. bryan and i pushed that directly into master earlier today

**Pull request merged**